src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -56,17 +57,6 @@ class DatePartFunctionNameSegment(BaseSegment):
     match_grammar = OneOf("DATEADD", "DATEDIFF")
 
 
-@redshift_dialect.segment(replace=True)
```

Comment:
Why was this moved to a different place in the file? It seems like an unnecessary change.

Reply:
I was originally going to organize the functions thematically and/or alphabetically but I didn't get that far.